swap/src/config.rs

```diff
@@ -13,7 +13,9 @@ use url::Url;
 
 pub mod seed;
 
-const DEFAULT_BITCOIND_TESTNET_URL: &str = "http://127.0.0.1:18332";
+const DEFAULT_ELECTRUM_HTTP_URL: &str = "https://blockstream.info/testnet/api/";
+// TODO: Not a sensible default, copied from https://github.com/bitcoindevkit/bdk
```

Comment:
Note: @rishflab tested using this URL in a test-project and it actually worked. Might want to remove the comment?